projects/ion/src/lib/input-select/input-select.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { SelectOption, ValueToEmmit } from '../core/types/input-select';
+
+const defaultSelectOptions: SelectOption[] = [
+  {
+    label: 'Maior que',
+  },
+  {
+    label: 'Entre',
+    multiple: true,
+    firstPlaceholder: 'Valor inicial',
+    secondPlaceholder: 'Valor final',
+  },
+  {
+    label: 'Igual a',
+  },
+  {
+    label: 'Maior ou igual a',
+  },
+  {
+    label: 'Menor que',
+  },
+  {
+    label: 'Menor ou igual a',
+  },
+];
+
+@Component({
+  selector: 'ion-input-select',
+  templateUrl: './input-select.component.html',
+  styleUrls: ['./input-select.component.scss'],
+})
+export class IonInputSelectComponent implements OnInit {
+  @Input() name?: string;
+  @Input() disabled = false;
+  @Input() value = '';
+  @Input() secondValue = '';
+  @Input() selectOptions: SelectOption[] = defaultSelectOptions;
+  @Output() valueChange = new EventEmitter<ValueToEmmit>();
+
+  public dropdownVisible = false;
+  public currentOption!: SelectOption;
+
+  public handleSelect(selectedOption: SelectOption[]): void {
```

Comment:
When select the option, it keeps the text secondValue.

Maybe he should clear both values when changing the option.

![image](https://github.com/Brisanet/ion/assets/15896994/c86b40f4-2182-4774-93f2-4d20afda4470)